tests/test_client.py

```diff
@@ -0,0 +1,330 @@
+import asyncio
+
+import pytest
+from aioresponses import CallbackResult, aioresponses
+
+from flexmeasures_client.client import FlexMeasuresClient
+
+
+def test__init__():
+    flexmeasures_localhost = FlexMeasuresClient("password", "email")
+    assert flexmeasures_localhost.__dict__ == {
```

Comment:
Nice test!

Reply:
Perhaps, it might be simplified by using [@pytest.mark.parametrize](https://docs.pytest.org/en/6.2.x/parametrize.html)